public/index.css

```diff
@@ -1,58 +0,0 @@
-/* note to self: it looks like this was styled by a child. fix it. */
-
```

Comment:
adding a `body { margin: 0 }` will remove the white border around the entire page, you can also shrink some of the content on the main page slightly to remove the small amount of scroll that is going on.